plugins/GSdx/GSState.cpp

```diff
@@ -809,6 +821,16 @@ void GSState::GIFRegHandlerXYZ2(const GIFReg* RESTRICT r)
 
 template<int i> void GSState::ApplyTEX0(GIFRegTEX0& TEX0)
 {
+	// Mask invalid bit before any comparison
+	TEX0.CPSM &= 0xa; // 1010b
+
+	if (PRIM->CTXT != i) {
+		// Context isn't current. Only update the register value.
+		// Handling (real write of the clut buffer) will be done when ctxt will be switched
+		m_env.CTXT[i].TEX0 = (GSVector4i)TEX0;
```

Comment:
????
Do you have more info  ? the callstack? I don't get it, the line was done below.